Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -439,7 +433,7 @@ fileprivate class BookmarkFolderTableViewHeader: UITableViewHeaderFooterView {
     lazy var chevron: ChevronView = {
         let chevron = ChevronView(direction: .left)
         chevron.tintColor = UIConstants.HighlightBlue
-        chevron.lineWidth = BookmarksPanelUX.BookmarkFolderChevronLineWidth
+        chevron.lineWidth = 2
```

Comment:
Nit: Use the constant, as declared. also:`BookmarkFolderChevronLineWidth` is 4.0, but you used 2.0.